Makefile

```diff
@@ -366,20 +366,14 @@ supplier_stake: ## Stake tokens for the supplier specified (must specify the SUP
 # TODO_IMPROVE(#180): Make sure genesis-staked actors are available via AccountKeeper
 .PHONY: supplier1_stake
 supplier1_stake: ## Stake supplier1 (also staked in genesis)
-	# TODO_UPNEXT(@okdas): once `relayminer` service is added to tilt, this hostname should point to that service.
```

Comment:
I checked configs @red-0ne introduced and didn't find a need to change anything to pass e2e tests.

Reply:
There's so much in flight that TODOs are going to get outdated quick. If you don't think it's relevant, remove it and we'll double check in code review!